bee-storage/bee-storage-rocksdb/src/access/insert.rs

```diff
@@ -44,3 +48,39 @@ impl Insert<Hash, MilestoneIndex> for Storage {
         Ok(())
     }
 }
+
+#[async_trait::async_trait]
+impl Insert<MessageId, Message> for Storage {
+    type Error = OpError;
+    async fn insert(&self, message_id: &MessageId, message: &Message) -> Result<(), Self::Error> {
+        let message_id_to_message = self.inner.cf_handle(MESSAGE_ID_TO_MESSAGE).unwrap();
+
+        let mut message_buf = Vec::new();
```

Comment:
Maybe prefer using `with_capacity` with the packed len, otherwise this is doing a lot of allocations internally